independent-publisher-connectors/DPIRD Weather/apiDefinition.swagger.json

```diff
@@ -0,0 +1,3611 @@
+        "description": "Returns a set of daily summaries for a specific station. The 24 hour period each field is summarised over varies between 24 hours to 9am (eg rainfall, minimum temperature), 24 hours from 9am (eg maximum temperature) and 24 hours from 12am (eg wind, solar).",
+        "operationId": "GetStationDailySummary",
+        "parameters": [{
+            "$ref": "#/parameters/stationCodeLookupInPath"
+          }, {
+            "$ref": "#/parameters/startDateImportantInQuery"
+          }, {
+            "$ref": "#/parameters/endDateImportantInQuery"
+          }, {
+            "$ref": "#/parameters/offsetInQuery"
+          }, {
+            "$ref": "#/parameters/limitInQuery"
+          }, {
+            "$ref": "#/parameters/sortInQuery"
+          }, {
+            "$ref": "#/parameters/selectInQuery"
+          }
+        ]
+      }
+    },
+    "/stations/{stationCode}/summaries/monthly": {
+      "get": {
+        "responses": {
+          "200": {
+            "description": "default",
+            "schema": {
+              "$ref": "#/definitions/singleStationSummarySchemaModel"
+            }
+          }
+        },
+        "summary": "Get Single Weather Station Monthly Summary",
+        "description": "Returns a set of monthly summaries for a specific station. The monthly period each field is summarised over is a monthly composite of the same 24 hour period used by the field in the daily summary. For example, rainfall, which uses the 24 hours to 9am daily period starts at 9am on the last day of the previous month and concludes at 9am on the last day of the month.",
+        "operationId": "GetStationMonthlySummary",
+        "parameters": [{
+            "$ref": "#/parameters/stationCodeLookupInPath"
+          }, {
+            "$ref": "#/parameters/startDateImportantInQuery"
+          }, {
+            "$ref": "#/parameters/endDateImportantInQuery"
+          }, {
+            "$ref": "#/parameters/offsetInQuery"
+          }, {
+            "$ref": "#/parameters/limitInQuery"
+          }, {
+            "$ref": "#/parameters/sortInQuery"
+          }, {
+            "$ref": "#/parameters/selectInQuery"
+          }
+        ]
+      }
+    },
+    "/stations/{stationCode}/summaries/yearly": {
+      "get": {
+        "responses": {
+          "200": {
+            "description": "default",
+            "schema": {
+              "$ref": "#/definitions/singleStationSummarySchemaModel"
+            }
+          }
+        },
+        "summary": "Get Single Weather Station Yearly Summary",
+        "description": "Returns a set of yearly summaries for a specific station. he yearly period each field is summarised over is a yearly composite of the same 24 hour period used by the field in the daily summary. For example, rainfall, which uses the 24 hours to 9am daily period starts at 9am on the last day of the previous year and concludes at 9am on the last day of the year.",

```

Comment:
All descriptions, summaries, titles etc. must be free from grammar errors. Please review them all and fix them.